src/pages/Delivery/Delivery.layout.jsx

```diff
@@ -66,12 +58,12 @@ const Delivery = ({
           >
             <img src={gps} width="22px" height="31px" alt="Gps icon" style={{ margin: '10px'}} />
           </FooterMapAddress>
-          <FooterMapAddress cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
-            <Text medium bold margin="8" style={{ textTransform: 'lowercase', color:'#003B62',fontWeight:'700'}}>
+          <FooterMapAddress style={{fontSize:'25px'}} cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
+            <Text medium bold margin="8" style={{ textTransform: 'lowercase',fontSize:'23px', color:'#003B62',fontWeight:'700'}}>
```

Comment:
please leave the proper space between each attribute, mm I noticed that sometimes it has space and sometimes not, please check what is the best standard for this case, or we could make a poll hehe.

I like more with the spaces